frontend/src/components/commons/AppLogo.vue

```diff
@@ -1,31 +1,52 @@
 <template>
   <v-footer class="custom-footer-logo" :light="false">
-    <v-sheet
-      v-for="(logo, $key) in logos"
-      :key="$key"
-      class="mx-auto px-4"
-      max-width="170"
-      min-width="150"
-    >
-      <a :href="logo.urlDescription" target="_blank" :aria-label="logo.label">
-        <v-img
-          :height="logo.height || '100px'"
-          contain
-          :src="logo.imgPath"
-        ></v-img>
-      </a>
-    </v-sheet>
+    <div class="logos-container align-center">
```

Comment:
why not directly in v-sheet ?

Reply:
Because I want to have the container to behave as a row